src/test/java/org/broadinstitute/hellbender/tools/walkers/qc/PileupIntegrationTest.java

```diff
@@ -0,0 +1,28 @@
+package org.broadinstitute.hellbender.tools.walkers.qc;
+
+import org.broadinstitute.hellbender.CommandLineProgramTest;
+import org.broadinstitute.hellbender.utils.test.IntegrationTestSpec;
+import org.testng.annotations.Test;
+
+import java.io.IOException;
+import java.util.Arrays;
+
+public class PileupIntegrationTest extends CommandLineProgramTest {
+	private static final String TEST_DATA_DIRECTORY = publicTestDir + "org/broadinstitute/hellbender/engine/";
+	private static final String TEST_OUTPUT_DIRECTORY = publicTestDir + "org/broadinstitute/hellbender/tools/walkers/qc/";
+
+    @Test
+    public void testPileup() throws IOException {
+		IntegrationTestSpec testSpec = new IntegrationTestSpec(
+			" -L 1" +
+				" -R " + hg19MiniReference +
+				" -I " + TEST_DATA_DIRECTORY + "reads_data_source_test1.bam" +
+				" -metadata " + TEST_DATA_DIRECTORY + "feature_data_source_test.vcf" +
+				" -outputInsertLength" +
+				" -O %s",
+			Arrays.asList(TEST_OUTPUT_DIRECTORY + "expectedPileup.txt")
```

Comment:
how was the expected output created? Did you use GATK3.5? Can you add a comment?